imports/deprecated/methods/communication/server/email.js

```diff
@@ -151,9 +150,9 @@ Meteor.methods({
 
         // this is a bug in core right now. We can't set Mandrill on the initial
         // post because it locks everything up, we can however, patch it
-        api.patch.sync(`Communications/${CommunicationId}`, {
-          MediumEntityTypeId: 37, // Mandrill
-        });
+        //api.patch.sync(`Communications/${CommunicationId}`, {
```

Comment:
Hey @cdanwards! If you put a space between the `//` and these lines of code, eslint (and therefore the build) will pass. You can see the failed build https://travis-ci.org/NewSpring/Holtzman/builds/316620523?utm_source=github_status&utm_medium=notification